content/php/concepts/arrays/terms/array-column/array-column.md

```diff
@@ -0,0 +1,120 @@
+---
+Title: 'array_column()'
+Description: 'Returns the values from a single column in the input array.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_column()`** function returns the values from a single column in the input array.
+
+## Syntax
+
+```pseudo
+array_column(array, column_key, index_key)
+```
+
+- `array`: A multi-dimensional array to use.
+- `column_key`: An integer or string key of the column of values to return. It may also be `null` to return complete arrays or objects.
+- `index_key`: The column to use as the index for the returned array.
+
+## Example
+
+In this example, the first names from an array are returned.
+
+```php
+<?php
+  $a = array(
+  array(
+    'id' => 362,
+    'first_name' => 'Jane',
+    'last_name' => 'Doe',
+  ),
+  array(
+    'id' => 921,
+    'first_name' => 'Lois',
+    'last_name' => 'Griffin',
+  ),
+  array(
+    'id' => 475,
+    'first_name' => 'Lola',
+    'last_name' => 'Bunny',
+  )
+);
+
+$first_names = array_column($a, 'first_name');
+print_r($first_names);
+?>
+```
+
+The output looks like this:
+
+```shell
+Array
+(
+  [0] => Jane
+  [1] => Lois
+  [2] => Lola
+)
+```
+
+Additionally, the returened values can be indexed by the "id" column:
+
+```php
+<?php
+  $a = array(
+  array(
+    'id' => 362,
+    'first_name' => 'Jane',
+    'last_name' => 'Doe',
+  ),
+  array(
+    'id' => 921,
+    'first_name' => 'Lois',
+    'last_name' => 'Griffin',
+  ),
+  array(
+    'id' => 475,
+    'first_name' => 'Lola',
+    'last_name' => 'Bunny',
+  )
+);
```

Comment:
I think we don't need to duplicate this part and we can just show the method with the new arguments